test/argent-account-test.ts

```diff
@@ -95,7 +96,8 @@ describe("Argent account", function () {
             const wrongKey = "0x0123";
             await starknet.ArgentAccount.getAccountFromAddress(argentAccountAddress, wrongKey);
             expect.fail("Should have failed on passing an incorrect private key.");
-        } catch (err: any) {
+        } catch (err) {
+            expect(err).to.be.instanceOf(HardhatPluginError);
```

Comment:
I think it would be better to have one common utility function used in every catch block. It would assert the type and error message:
```typescript
   expectStarknetPluginError(err, expectedMessage);
```

This would reduce line count, but doesn't solve the `unknown` type problem.

Reply:
i added 3 helpers :
`expectStarknetPluginError(err)` // check type
`expectStarknetPluginErrorEqual(err, expectedMessage)` // check type + message equal
`expectStarknetPluginErrorContain(err, expectedMessage)` // check type + message contain